.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
This forces the user to use one of the templates and does not longer allow blank issues.

Reply:
I often open blank issues when the issue is simple and known. Better than just keeping notes privately.